draft-ietf-lake-edhoc.md

```diff
@@ -805,7 +828,7 @@ message_2 SHALL be a CBOR Sequence (see {{CBOR}}) as defined below
 ~~~~~~~~~~~ CDDL
 message_2 = (
   G_Y_CIPHERTEXT_2 : bstr,
-  C_R : bstr / int,
+  C_R : bstr,
```

Comment:
Also consistently with the later editing in the "CDDL Definitions" appendix, this should be

`C_R : bstr / -24..23,`

Reply:
Overtaken by later proposals